src/schema/landingPage/resolvers.ts

```diff
@@ -7,27 +7,30 @@ const normalizeLandingPage = (collection) => {
   let normalizedLandingPage = normalizeKeys(collection);
   const normalizedKeys = [
     'title',
+
+    // BEGIN deprecated fields
```

Comment:
When these will be removed? Perhaps after these changes are in use in events-helsinki-ui?

Reply:
Yes, this will be removed after CMS API completed data transfer process (i.e bottom banner added to the API completely, old banner data moved to new top banner data)

Reply:
You code use `TODO` or `FIXME` instead of `BEGIN` as it will be treated as annotation by some Ides like vscode and intellij.

Reply:
Oops, sorry I missed your comment here. I will fix the annotation in the next PR 😔